examples/sample-hardhat-project/cannonfile.consumer.toml

```diff
@@ -15,7 +15,7 @@ options.salt = "second"
 options.msg = "a message from second greeter set"
 
 [invoke.do_change_greeting2]
-target = ["greeters.Greeter"]
+target = ["greeters.Greeterz"]
```

Comment:
Is this ok?

Reply:
i was testing something; probably should be reverted back though